packages/gatsby-remark-images/src/__tests__/index.js

```diff
@@ -68,7 +67,7 @@ const createPluginOptions = (content, imagePaths = `/`) => {
   return {
     files: [].concat(imagePaths).map(imagePath => {
       return {
-        absolutePath: queryString.parseUrl(`${dirName}/${imagePath}`).url,
+        absolutePath: `${dirName}/${imagePath.split('?')[0]}`,
```

Comment:
I don't know what this piece of code is doing since it's not a real URL. My assumption is that `imagePath` might contain a `?some=thing` string so I'm simply dropping that